model/flow/identifier.go

```diff
@@ -196,7 +197,7 @@ func Sample(size uint, ids ...Identifier) []Identifier {
 	return dup[:size]
 }
 
-func CidToFlowID(c cid.Cid) (Identifier, error) {
+func CidToId(c cid.Cid) (Identifier, error) {
```

Comment:
is there a reason for this rename? `CidToFlowID` seems less confusing, no?

Reply:
It is externally used as `flow.CiDToID(..)`, and since it has the package name, I found the `Flow` in the function name redundant. But I can take it back if you think it is confusing.

Reply:
no that makes sense.